server/pulp/server/db/manage.py

```diff
@@ -191,6 +193,14 @@ def main():
         options = parse_args()
         _start_logging()
         connection.initialize(max_timeout=1)
+
+        # Prompt the user if there are active workers or running tasks
+        if status.get_workers() or filter(lambda x: x.state in constants.CALL_INCOMPLETE_STATES,
```

Comment:
I think we should disinclude the check for tasks in the incomplete states. The only unsafe aspect really is if workers are running. Sometimes bugs put task states incorrectly in an incomplete state so I don't want those bugs to have an impact on this functionality.